publication-request.json

```diff
@@ -0,0 +1,16 @@
+{
+    "package-id": "hl7.fhir.uv.cgm",
+    "version": "1.0.0-ballot",
+    "path": "http://hl7.org/fhir/uv/cgm/2025May",
+    "mode": "working",
```

Comment:
Should this be `milestone` if this is considered to be a version that stays available in version history? Not sure.